python/ray/train/lightning/lightning_trainer.py

```diff
@@ -131,8 +148,19 @@ def ddp_strategy(self, **kwargs) -> "LightningConfigBuilder":
     def checkpointing(self, **kwargs) -> "LightningConfigBuilder":
         """Set up the configurations of ``pytorch_lightning.callbacks.ModelCheckpoint``.
 
-        LightningTrainer creates a `ModelCheckpoint` callback based on this config.
-        The AIR checkpointing and logging methods are triggered in that callback.
+        LightningTrainer creates a subclass instance of the `ModelCheckpoint` callback
+        with the kwargs. It handles checkpointing and metrics logging logics.
+
+        Specifically, the callback periodically reports the latest metrics and
+        checkpoint to the AIR session via `session.report()`. The report frequency
+        matches the checkpointing frequency here. You have to make sure that the
+        target metrics (e.g. metrics defined in `TuneConfig` or `CheckpointConfig`)
```

Comment:
Same here, ideally use `ref` to reference the API docs

Reply:
Done!